### PR TITLE
~/.show-rss timestamp is latest download time

### PR DESCRIPTION
**Pull request by catkfr**

### CODE REVIEW

**Comment by 0x0L** [comment]
Can you fix the indentation around lines 40+ and 90+ and squash the commits ?
Pls make sure to indent using spaces. Thanks.

File: show-rss-helper.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 from urllib import urlencode
 from urllib2 import build_opener
 from xml.dom import minidom
+import time
```
Comment (0x0L) [inline show-rss-helper.py]: only import what's needed

``` python
from time import mktime
```

File: show-rss-helper.py [hunk 2]
```diff
@@ -75,9 +79,18 @@ def parse_dates_links(feed):
 
 if __name__ == '__main__':
     timestamp = get_timestamp()
+    latest = None
+    odate = None
+    if timestamp is not None:
```
Comment (0x0L) [inline show-rss-helper.py]: `if timestamp:`

**Comment by catkfr** [comment]
comments taken into consideration

**Comment by 0x0L** [comment]
Thanks

**Pull request merged**